crates/basilica-sdk-python/python/basilica/distributed.py

```
@@ -1060,6 +1062,120 @@ def _derive_rendezvous_endpoint(self) -> str:
+        self,
+        exc_type: Optional[Type[BaseException]],
+        exc_val: Optional[BaseException],
+        exc_tb: Optional[TracebackType],
+    ) -> None:
+        # Best-effort cleanup. Swallow any delete-time exception so we
+        # never mask the (possibly more important) exception that is
+        # already propagating out of the `with` block.
+        try:
+            self._training.delete()
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)